cypress/integration/board.spec.js

```diff
@@ -0,0 +1,61 @@
+import createOrganization from "../fixtures/createOrganization.json"
+import loginPage from "../fixtures/loginPage.json"
+import data from "../fixtures/data.json"
+import navigation from "../fixtures/navigation.json"
+import myOrganizations from "../fixtures/myOrganizations.json"
+import createBoard from "../fixtures/createBoard.json"
+
+describe("CRUD Board", ()=>{
```

Comment:
Please try to write

```suggestion
describe("CRUD Board", () => {
```

wherever applicable.